yaml-tests/src/test/java/YamlIntegrationTests.java

```diff
@@ -18,232 +18,237 @@
  * limitations under the License.
  */
 
+import com.apple.foundationdb.relational.yamltests.ExcludeYamlTestConfig;
+import com.apple.foundationdb.relational.yamltests.YamlTest;
+import com.apple.foundationdb.relational.yamltests.configs.JDBCInProcessConfig;
 import org.junit.jupiter.api.Disabled;
-import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestTemplate;
 
 /**
- * Base class covering all the integration tests specified by yamsql files.
+ * Class covering the standard integration tests specified by yamsql files.
  */
-public abstract class YamlIntegrationTests extends YamlTestBase {
-    @Test
-    public void showcasingTests() throws Exception {
-        doRun("showcasing-tests.yamsql");
+@YamlTest
+public class YamlIntegrationTests {
+    @TestTemplate
+    public void showcasingTests(YamlTest.Runner runner) throws Exception {
+        runner.run("showcasing-tests.yamsql");
     }
 
-    @Test
-    public void groupByTests() throws Exception {
-        doRun("groupby-tests.yamsql");
+    @TestTemplate
+    public void groupByTests(YamlTest.Runner runner) throws Exception {
+        runner.run("groupby-tests.yamsql");
     }
 
-    @Test
-    public void standardTests() throws Exception {
-        doRun("standard-tests.yamsql");
+    @TestTemplate
+    @ExcludeYamlTestConfig(JDBCInProcessConfig.class)
```

Comment:
Maybe add a reason for the exclusion (similar to `@Disabled`)?

Reply:
1. This was actually accidentally disabled, so I removed this
2. I added a `reason` field to the annotation, and filled it out with the new ones added for JDBC
3. At some point we may need to deal with having multiple configs, as they would probably have different reasons. Repeating the annotation, and changing it to not have an array for the values. But, I think it makes sense to figure that out when we have an opportunity.